.github/workflows/cherry-pick.yml

```diff
@@ -0,0 +1,57 @@
+name: Create a cherry pick from a PR
+
+on:
+  repository_dispatch:
+    types: [try-cherry-pick]
+
+jobs:
+  cherry-pick:
+    name: cherry-pick-pr-${{ github.event.client_payload.pr_num }}
+    runs-on: ubuntu-latest
+    environment: cherry-pick-bot
```

Comment:
Just FYI, we will need to create a new env with this name and add the `GH_PYTORCHBOT_CHERRY_PICK_TOKEN`.  Let me add it now.

Reply:
The env is ready https://github.com/pytorch/executorch/settings/environments/2551100531

Reply:
> we will need to create a new env with this name and add the GH_PYTORCHBOT_CHERRY_PICK_TOKEN. 

Aren't we using the same pytorchbot which should has the token already?

> The env is ready https://github.com/pytorch/executorch/settings/environments/2551100531
@huydhn  BTW, I'm not able to access the link. Got 404 error.

Reply:
From my notes, I need to create a new token with one more permission to create PR to update the workflow files in addition to regular files like the pinned commits